src/rpc/grpc.md

```diff
@@ -0,0 +1,40 @@
+# gRPC
+
+Sitting on top of HTTP/2 and protobuf gRPC provides a high performance RPC

```

Comment:
```suggestion
Sitting on top of HTTP/2 and protobuf, gRPC provides a high-performance RPC
```

Reply:
```suggestion
Sitting on top of HTTP/2 and protocol buffers gRPC provides a high performance RPC
```

I would also add links for http/2 and protobufs.

Should we summarise what RPC is?

Reply:
Ah, I see there is an RPC intro before this, ignore the second comment